### PR TITLE
Dashboard_look_made_Same_As_EvalAI

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
Feature:
Dashboard look of EvalAI-ngx is made same as that of EValAI.

GIF:
ezgif com-video-to-gif

### CODE REVIEW

File: src/app/components/dashboard/dashboard-content/dashboard-content.component.scss [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
+  width: 300px;
+}
+.btn{
+  font-size: 1.2rem;
+  padding-top:.5%;
+  height: 45px;
+}
+.text-dark-black{
+  font-size: 3rem;
 }
```
Comment (Suryansh5545) [inline src/app/components/dashboard/dashboard-content/dashboard-content.component.scss:17]: Please Give a space

**Review by Suryansh5545 — CHANGES_REQUESTED** [review]

File: src/app/components/dashboard/dashboard-content/dashboard-content.component.scss (hunk 1 above)
Comment (Kajol-Kumari) [reply src/app/components/dashboard/dashboard-content/dashboard-content.component.scss:17]: > Please Give a space

@Suryansh5545 are you taking about giving one extra line after the last line? 'coz i have already given it but i don't know why it is not getting reflected here.

**Review by Shekharrajak — COMMENTED** [review]
Please check the responsiveness of the UI after this change for different devices (mobile, tablet, PC). Try to use css from the materialize framework which is already included.

File: src/app/components/dashboard/dashboard-content/dashboard-content.component.scss (hunk 1 above)
Comment (Suryansh5545) [reply src/app/components/dashboard/dashboard-content/dashboard-content.component.scss:17]: @kajol-kumari if it is not showing then edit it in github and pull the commit to your local branch

Comment (Kajol-Kumari) [reply src/app/components/dashboard/dashboard-content/dashboard-content.component.scss:17]: > @Kajol-Kumari if it is not showing then edit it in github and pull the commit to your local branch

@Suryansh5545 the extra line is already in my github "dashboard_look_same_As_EvalAI" branch. It's just not getting reflected here.

Comment (Kajol-Kumari) [reply src/app/components/dashboard/dashboard-content/dashboard-content.component.scss:17]: @Suryansh5545 I just tried adding two extra blank line in .scss file but it is showing only one here.Should i keep it like this?

**Comment by Suryansh5545** [comment]
@Kajol-Kumari instead of pushing do a change in github only.

**Comment by Kajol-Kumari** [comment]
> @Kajol-Kumari instead of pushing do a change in github only.

@Suryansh5545 What changes you are taking about as the github already contains one extra blank line. When i added two extra line in the code and pushed it, it is showing a single line in both my branch as well as here so i think i can  keep it, what you say?

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**